src/Behat/Behat/Output/Node/Printer/JUnit/JUnitScenarioPrinter.php

```diff
@@ -85,29 +65,42 @@ public function printOpenTag(Formatter $formatter, FeatureNode $feature, Scenari
 
         if ($file) {
             $cwd = realpath(getcwd());
-            $testCaseAttributes['file'] =
-                substr($file, 0, strlen($cwd)) === $cwd ?
-                    ltrim(substr($file, strlen($cwd)), DIRECTORY_SEPARATOR) : $file;
+            $testCaseAttributes['file'] = substr($file, 0, strlen($cwd)) === $cwd
+                ? ltrim(substr($file, strlen($cwd)), DIRECTORY_SEPARATOR)
+                : $file;
         }
 
         $outputPrinter->addTestcase($testCaseAttributes);
     }
 
-    /**
-     * @param ExampleNode $scenario
-     * @return string
-     */
-    private function buildExampleName(ExampleNode $scenario)
+    private function buildExampleName(ExampleNode $scenario): string
```

Comment:
I'm wondering if it would make sense to have this function inside `ExampleNode` - it might be useful elsewhere.

I have implemented something similar for a custom report at work, for example.

Reply:
Relates to the title discussion, so this can be closed.